tests/system/providers/cohere/example_cohere_embedding_operator.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.decorators import task
+from airflow.providers.cohere.operators.embedding import CohereEmbeddingOperator
+
+with DAG("example_cohere_embedding", schedule=None, start_date=datetime(2023, 1, 1), catchup=False) as dag:
```

Comment:
```suggestion
with DAG("example_cohere_embedding", schedule=None, start_date=datetime(2023, 1, 1), catchup=False):
```
No longer needed after Airflow 2.4.0.

Reply:
@josh-fell We later require it for below code. Referenced it from existing code - https://github.com/apache/airflow/blob/3724a029dadf4678d1cc89049b247f6d2bc233e2/tests/system/providers/slack/example_sql_to_slack.py#L56 But I'm happy to change if there is workaround.
```
from tests.system.utils import get_test_run

# Needed to run the example DAG with pytest (see: tests/system/README.md#run_via_pytest)
test_run = get_test_run(dag)
```

Reply:
Ah yes my mistake.